FinniversKit/Sources/Recycling/GridViews/Markets/GridView/Cell/MarketsGridViewCell.swift

```diff
@@ -8,7 +8,9 @@ class MarketsGridViewCell: UICollectionViewCell {
     // MARK: - Internal properties
 
     private let cornerRadius: CGFloat = 16
-
+    
+    var isFinn: Bool = false //
```

Comment:
i've ran the demo app with `isFinn = true `

the view looks like this:
<img width="400" alt="image" src="https://github.com/finn-no/FinniversKit/assets/23578783/f89adcb2-4543-4b65-a7da-d86c837de865">

I think it's best to keep the current behaviour of finn:
<img width="400" alt="image" src="https://github.com/finn-no/FinniversKit/assets/23578783/b66e09b9-1e65-45aa-a5fb-16c8405989df">